contracts/pricers/ChainlinkPricer.sol

```diff
@@ -45,8 +47,8 @@ contract ChainLinkPricer is OpynPricerInterface {
     function getPrice() external override view returns (uint256) {
         int256 answer = aggregator.latestAnswer();
         require(answer > 0, "ChainLinkPricer: price is lower than 0");
-        // Scaled to 1e18 from 1e8
-        return uint256(answer * 1e10);
+        // Scale to 1e18 from 1e8
+        return uint256(answer).mul(BASE);
```

Comment:
Can we start using the FixedPoint to represent any scaled variable ? instead of uint256

Reply:
+1

Reply:
Is the chainLink on chain prices 1e8?

Reply:
don't think we need to because our FixPoint is int256.
Also that will change everything in Oracle, can do that in a separate PR as improvement